documentation/manual/prereq.rst

```diff
@@ -153,16 +153,15 @@ example:
 
 .. code-block:: bash
 
-    $ wget -O - https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.10/hdf5-1.10.2/src/hdf5-1.10.2.tar.gz | tar xvzf -
-    $ cd hdf5-1.10.2
-    $ ./configure --prefix=$HOME/local --enable-fortran --enable-fortran2003 --enable-cxx
+    $ wget -O - https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.10/hdf5-1.10.4/src/hdf5-1.10.4.tar.gz | tar xvzf -
+    $ cd hdf5-1.10.4
+    $ ./configure --prefix=$HOME/local --enable-fortran --enable-cxx
```

Comment:
Nice catch. Is `--enable-cxx` truly needed?

Reply:
It appears not!